transforms3d/tests/test_quaternions.py

```diff
@@ -104,6 +104,15 @@ def test_qeye():
     assert np.all([1,0,0,0]==qi)
     assert np.allclose(tq.quat2mat(qi), np.eye(3))
 
+def test_qexp():
+    angular_velocity_pure_quaterion = np.array([0., math.pi, 0, 0])
+    dt = 1.0
+    q_integrate_angular_vel = tq.qexp(angular_velocity_pure_quaterion * dt/2)
+    #see https://www.ashwinnarayan.com/post/how-to-integrate-quaternions/ near the end. 
```

Comment:
Would you mind putting a version of these notes in the docstring? I see the formula for the exponential is in the Quaternion wikipedia page:

https://en.wikipedia.org/wiki/Quaternion#Exponential,_logarithm,_and_power

and there's a derivation here:

https://math.stackexchange.com/questions/1030737/exponential-function-of-quaternion-derivation

Maybe cite these for clarity?

I know this is a big ask, but would you consider calculating quaternion exponential for quaternions in `quats` variable in test module, and compare against results from some other software?